pkg/webhook/validator_result.go

```diff
@@ -33,6 +37,10 @@ type PolicyResult struct {
 	// AuthorityMatches will have an entry for each successful Authority check
 	// on it. Key in the map is the Attestation.Name
 	AuthorityMatches map[string]AuthorityMatch `json:"authorityMatches,omitempty"`
+
+	// Config contains the Config for each of the architectures where key
+	// to the map is the architecture.
```

Comment:
Is it the architecture or the serialized platform/architecture string?

e.g. could this be `linux/arm/v6`?